README.md

```diff
@@ -1,33 +1,34 @@
-# Uppy
+# [Uppy](http://uppy.io)
 
-<img src="http://uppy.io/images/logos/uppy-dog-full.svg" width="120" alt="Uppy logo — a superman puppy in a pink suit" align="right">
+<img src="http://uppy.io/images/logos/uppy-dog-head-arrow.svg" width="120" alt="Uppy logo — a superman puppy in a pink suit" align="right">
 
-<a href="https://www.npmjs.com/package/uppy"><img src="https://img.shields.io/badge/stability-experimental-orange.svg?style=flat-square"></a>
 <a href="https://www.npmjs.com/package/uppy"><img src="https://img.shields.io/npm/v/uppy.svg?style=flat-square"></a>
 <a href="https://travis-ci.org/transloadit/uppy"><img src="https://img.shields.io/travis/transloadit/uppy/master.svg?style=flat-square" alt="Build Status"></a>
+<a href="https://www.npmjs.com/package/uppy"><img src="https://img.shields.io/badge/stability-experimental-orange.svg?style=flat-square"></a>
 
-Uppy is a sleek, modular file uploader that integrates seemlessly with any framework. It fetches files from local disk, Google Drive, Dropbox, Instagram, remote URLs, cameras and other exciting locations, and then uploads them to the final destination. It’s fast, easy to use and let's you worry about more important problems than building a file uploader. [Try it live](http://uppy.io/examples/dashboard/).
+Uppy is a sleek, modular file uploader that integrates seemlessly with any application. It’s fast, easy to use and lets you worry about more important problems than building a file uploader.
 
-Uppy is developed by the [Transloadit](https://transloadit.com) team.
+- **Fetch** files from local disk, Google Drive, Dropbox, Instagram, or take selfies and videos with a camera;
+- **Preview** and edit metadata with a nice interface;
+- **Upload** to the final destination, optionally processing/encoding on the way;
 
-Check out docs and examples on [uppy.io](http://uppy.io).
+**[Try Uppy](http://uppy.io/examples/dashboard/)** | **[Read the docs](http://uppy.io/docs)**
 
 <img width="700" alt="Uppy UI Demo: modal dialog with a few selected files and an upload button" src="https://github.com/transloadit/uppy/raw/master/uppy-screenshot.jpg">
 
-- [Full featured UI](http://uppy.io/examples/dashboard)
-- [Simple Drag & Drop](http://uppy.io/examples/dragdrop)
+Uppy is being developed by the [Transloadit](https://transloadit.com) team.
 
 ## Features
 
 - Lightweight, modular plugin-based architecture, easy on dependencies :zap:
-- Use from a CDN or as a module to import
```

Comment:
Is this no longer a goal? 😱 or just not something to advertise on the intro page?

Reply:
I guess this information is one paragraph lower down in the readme, nvm :D